### ISSUE TITLE
TR1 healthbar lingers for too long in the inventory

### ISSUE THREAD

**Issue by rr-** [LostArtefacts/TRX#1991]
Found in #1983 for TR2, also present in TR1.

- [ ] In the inventory, when the medi pack is highlighted and you switch to another item, the health bar persists until the next item is highlighted. In OG, the health bar would disappear the moment you press left/right to switch.
- [ ] Similar issue to above, if you have a medi pack highlighted when switching to rings above or below, the health bar is slow to disappear and quick to reappear. In OG, the health bar would appear/disappear at the same time as the green arrows in the middle ring.

@aredfan this is a very old TR1X bug. Can I ask for pinpointing the version that broke this? Thanks.

**Comment by aredfan** [comment]
@rr- Of course!  This regression first appeared in [4.0](https://github.com/LostArtefacts/TRX/releases/tag/tr1-4.0).
